### ISSUE TITLE
Quite different output compared to ADMIXTURE

### ISSUE THREAD

**Issue by privefl** [AI-sandbox/neural-admixture#8]
### With ADMIXTURE (K = 15):
![admixture](https://user-images.githubusercontent.com/11161036/138671568-bdb4695a-3e18-41dc-ac1b-728900e99967.jpg)

### With Neural ADMIXTURE (K = 15):
![neural-admixture](https://user-images.githubusercontent.com/11161036/138671683-c0c58a2c-8ca2-47c2-b8ce-95ea5118a7af.jpg)

***

Do you have any explanation on why I get these large differences?
What should I try? Using more PCs in the initialization?

**Comment by AlbertDominguez** [comment]
Hi,

It looks like in this case Neural ADMIXTURE is able to capture the structure of the super-population but fails to separate inside these superpopulations. I noticed this was done with an older version of the code. I recommend you update the program to the latest version available (which we release today) and try the _PCArchetypal_ analysis, which might perform better in this scenario. The corresponding article update explaining this initialization method will be available soon. We've also changed some default values and introduced some changes in the architecture in order to improve it.

I also suggest you try running the multi-head version (specifying `--min_k` and `--max_k` instead of `--k`) and check results for several values of K and see how assignments evolve as K grows.

Moreover, we believe the number of PCs used at initialization does not critically impact performance. Playing with other hyperparameters, such as the learning rate, does have a greater impact than the number of PCs.

Finally, just mention that results may quite differ from ADMIXTURE. We already have in mind providing a framework to allow easy sweep runs in order to make trying several hyperparameter values combinations and inspecting the results easier, this hopefully makes everything easier and automates things a bit!
